### PR TITLE
[CELEBORN-1028] Make prometheus path configurable

### PR DESCRIPTION
**Pull request by cxzl25**
### What changes were proposed in this pull request?
`celeborn.metrics.prometheus.path`

### CODE REVIEW

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala [hunk 1]
```diff
@@ -3661,6 +3661,14 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(10)
 
+  val METRICS_PROMETHEUS_PATH: ConfigEntry[String] =
+    buildConf("celeborn.metrics.prometheus.path")
+      .categories("metrics")
+      .doc("URI context path of prometheus metrics HTTP server")
```
Comment (waitinfuture) [inline common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:3667]: better to add version for the config

**Comment by xleoken** [comment]
Hi @cxzl25, it doesn't seem necessary to make such a change, can you provide a practical scenario? thanks.

**Comment by cxzl25** [comment]
> it doesn't seem necessary to make such a change, can you provide a practical scenario? 

Spark and Kyuubi support configuring Prometheus Servlet to register on different paths.

https://github.com/apache/spark/blob/08640961e3bad7de38ed3358df8706bad028c27a/core/src/main/scala/org/apache/spark/metrics/sink/PrometheusServlet.scala#L41-L45

https://github.com/apache/kyuubi/blob/e33df9ce6664b7069baceaf3b5a874464ea87cd7/kyuubi-metrics/src/main/scala/org/apache/kyuubi/metrics/PrometheusReporterService.scala#L42

**Comment by xleoken** [comment]
> > it doesn't seem necessary to make such a change, can you provide a practical scenario?
> 
> Spark and Kyuubi support configuring Prometheus Servlet to register on different paths.
> 
> https://github.com/apache/spark/blob/08640961e3bad7de38ed3358df8706bad028c27a/core/src/main/scala/org/apache/spark/metrics/sink/PrometheusServlet.scala#L41-L45
> 
> https://github.com/apache/kyuubi/blob/e33df9ce6664b7069baceaf3b5a874464ea87cd7/kyuubi-metrics/src/main/scala/org/apache/kyuubi/metrics/PrometheusReporterService.scala#L42

thanks, it seems not necessary from my side 😅

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks! Merging to main.

**Pull request closed without merging**